UefiTestingPkg/FunctionalSystemTests/MemoryProtectionTest/Readme.md

```diff
@@ -7,15 +7,46 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 ## About This Test

```

Comment:
For the times given in this section, can you briefly describe the environment you're referencing? For example, "Estimated run times in this section are based on a QEMU DEBUG build".

Reply:
Added a note that the benchmark is Q35 debug build